src/Provider/CustomerProviderInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\AdminOrderCreationPlugin\Provider;
+
+use Sylius\Component\Core\Model\CustomerInterface;
+
+interface CustomerProviderInterface
+{
+    /**
+     * Loads an existing customer from the database and throws an error if it does not exist
+     *
+     * @param string $id
+     *
+     * @return CustomerInterface
+     */
+    public function provideExistingCustomer(string $id): CustomerInterface;
```

Comment:
And why `$id` is `string`? It should be `int`, I suppose

Reply:
The idea behind this was that the Ids can also be UUIDs and therefore strings.